CodeGeneration/Sources/Utils/CodeGenerationFormat.swift

```diff
@@ -42,4 +42,71 @@ public class CodeGenerationFormat: BasicFormat {
       return super.visit(node)
     }
   }
+
+  public override func visit(_ node: TupleExprElementListSyntax) -> TupleExprElementListSyntax {
+    let children = node.children(viewMode: .all)
+    // If the function only takes a single argument, display it on the same line
+    if let callee = node.parent?.as(FunctionCallExprSyntax.self)?.calledExpression.as(MemberAccessExprSyntax.self), callee.base == nil {
+      // This is a constructor for tokens. Write them on a single line
+      return super.visit(node)
+    }
```

Comment:
Do we need this or did you just copy it from the `debugInitCall` format?

Reply:
Not needed. Just took it from the `debugInitCall ` format